contracts/collection.scilla

```diff
@@ -609,3 +641,26 @@ transition ClearAllowList()
   };
   event e
 end
+
+(* @multi-sig *)
+transition SetFixedPrice(address: ByStr20)
```

Comment:
@sam-zilliqa is it possible to support multiple fixed price contracts? Because it would be possible for Rialto to have multiple fixed price contracts (i.e. when we deploy a new one and we're slowly phasing out the previous contract).

Reply:
Checking if asset is listed in FP or auction is removed, this is done offchain